src/views/AddItem.jsx

```diff
@@ -109,57 +122,113 @@ export function AddItem({ listToken, data }) {
 
 	return (
 		<>
-			<Form onSubmit={onFormSubmit}>
-				<Form.Label htmlFor="item">Item Name:</Form.Label>
-				<InputGroup>
-					<Form.Control
-						type="text"
-						id="item"
-						name="item"
-						value={itemName}
-						onChange={(e) => setItemName(e.target.value)}
-						required={true}
-					/>
-				</InputGroup>
-				<Form.Label id="purchase-date-label">
-					How soon will you buy this again?
-				</Form.Label>
-				<fieldset>
-					<div>
-						<Form.Check
-							defaultChecked
-							aria-labelledby="purchase-date-label"
-							type="radio"
-							value="soon"
-							id="soon"
-							name="radio-btn"
-							onChange={handlePurchaseDate}
-							label="Soon"
-						/>
-					</div>
-					<div>
-						<Form.Check
-							type="radio"
-							value="kind-of-soon"
-							id="kind-of-soon"
-							name="radio-btn"
-							onChange={handlePurchaseDate}
-							label="Kind of Soon"
-						/>
-					</div>
-					<div>
-						<Form.Check
-							type="radio"
-							id="not-soon"
-							value="not-soon"
-							name="radio-btn"
-							onChange={handlePurchaseDate}
-							label="Not Soon"
-						/>
-					</div>
-				</fieldset>
-				<Button type="submit">Add Item</Button>
-			</Form>
+			<div className="add-item d-flex flex-column">
+				<header className="text-center">
+					<h1>Add Item</h1>
+				</header>
+
+				<Container className="add-item-form shadow rounded-3 p-5 mt-5 bg-white">
+					<Form onSubmit={onFormSubmit}>
+						<div className="pb-4 mx-5">
+							<Form.Label htmlFor="item">Item Name:</Form.Label>
+							<InputGroup>
+								<Form.Control
```

Comment:
Looking ahead at consistency for search inputs, I know that the search bar for the list page is different but I think we can consolidate in the consistent styling issue. But great work because this looks exactly like the wireframe!

Reply:
Good point! It looks like the styling in the wireframe was a little different, but it's definitely worth thinking through whether we want them to look the same!